.github/workflows/pipeline.yaml

```
@@ -0,0 +1,16 @@
+name: Pipeline
+
+on:
+  push:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  deliver:
+    uses: ./.github/workflows/delivery.yaml
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/delivery.yaml": open /home/jailuser/git/.github/workflows/delivery.yaml: no such file or directory

(workflow-call)